modules/calib3d/test/test_stereomatching.cpp

```diff
@@ -791,6 +791,11 @@ class CV_StereoBMTest : public CV_StereoMatchingTest
         bm->compute( leftImg, rightImg, tempDisp );
         tempDisp.convertTo(leftDisp, CV_32F, 1./StereoMatcher::DISP_SCALE);
 
+        //check for fixed-type disparity data type
+        Mat_<float> fixedFloatDisp;
+        bm->compute( leftImg, rightImg, fixedFloatDisp );
+        EXPECT_TRUE(cvtest::norm(fixedFloatDisp, leftDisp, cv::NORM_L2) < (double)(leftDisp.total()));
```

Comment:
1. Use **relative** norm L2.
2. Use threshold something like 1e-3.

Reply:
Threshold of such scale seems unreachable since it's a difference between float-point and short-int calculations. The typical measured (relative L2) error is 0.9375.

Reply:
> The typical measured (relative L2) error is 0.9375

This means that compared data is almost **not** similar.
So you just can not apply this check to verify corectness.


Dump these values:
```
std::cout << "norm x256=" << cvtest::norm(fixedFloatDisp * 256, Mat_<float>(tempDisp), cv::NORM_L2 | cv::NORM_RELATIVE, mask) << std::endl;
std::cout << "norm x16=" << cvtest::norm(fixedFloatDisp * 16, Mat_<float>(tempDisp), cv::NORM_L2 | cv::NORM_RELATIVE, mask) << std::endl;
```

16 is `StereoMatcher::DISP_SCALE`, but what is 256? Bug?